docs/developer/eventing/sources/kafka-source/README.md

```diff
@@ -148,19 +146,17 @@ The `KafkaSource` reads all the messages, from all partitions, and sends those m
     ```
 
     Example output:
-
     ```bash
     kafkasource.sources.knative.dev/kafka-source created
     ```
 
-1. Check that the event source pod is running:
+1. Verify that the event source Pod is running:
 
     ```bash
     kubectl get pods
     ```
 
-    The pod name will be prefixed with `kafka-source`:
-
+    The Pod name is prefixed with `kafka-source`:
     ```bash
     NAME                                  READY     STATUS    RESTARTS   AGE
     kafka-source-xlnhq-5544766765-dnl5s   1/1       Running   0          40m
```

Comment:
Not sure whether you need to run the `kubectl logs` command on line 169 as well as line 170. Or is line 169 a command and 170 the output?

Reply:
Can you answer this, @matzew?

Reply:
169 is the command, 170 is an example output

Reply:
Thanks, @lberk.